frontend/src/components/DecayInformations/DecayInformation-Short.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="decayinformation-short">
+    <span>− {{ decay ? $n(Number(decay.decay) * -1, 'decimal') : '' }}</span>
+  </div>
+</template>
+<script>
+export default {
+  name: 'DecayInformation-Short',
+  props: {
+    decay: {
+      type: Object,
```

Comment:
Would be cleaner to have decay as a Number props that is given from the outside instead of casting it in the HTML